dojo/tools/vulners/api_client.py

```diff
@@ -0,0 +1,33 @@
+import vulners
+
+
+class VulnersAPI:
+    """
+    A simple client for the Vulners API
+    """
+
+    DEFAULT_URL = "https://vulners.com"
+    vulners_api_url = ''
+    api_key = ''
+
+    def __init__(self, tool_config):
+        if tool_config.authentication_type == "API":
+            self.api_key = tool_config.api_key
+            self.vulners_api_url = tool_config.url or self.DEFAULT_URL
+        else:
+            raise Exception('Vulners.com Authentication type {} not supported'.format(tool_config.authentication_type))
+
+    def get_client(self):
+        return vulners.VulnersApi(
+            api_key=self.api_key,
+            server_url=self.vulners_api_url,
+            persistent=False
```

Comment:
why this? forcing to false make creating a connection for each call to the API

Reply:
also `server_url` already have a default value in the lib, why forcing it?

(from the 2.0.4)
```Python
def __init__(
        self, api_key, proxies=None, persistent=True, server_url="https://vulners.com"
    ):
        """
        Create VScanner API object.

        :param api_key:
            string with Vulners API key. You can obtain one from the https://vulners.com
        :param proxies:
            dict for proxy supporting. Example: {"https": "myproxy.com:3128"}
        """
```

Also, regarding our discussion, I was expecting thta you needed the support for proxies.
Something like:
```
    def get_client(self):
        return vulners.VulnersApi(
            api_key=self.api_key,
            proxies=self.proxies,  # ex: {"https": "myproxy.com:3128"}
```

Reply:
> This is better but the current PR doesn't support proxies. is it still one of your goal?

Proxy in this PR is not exactly a default http proxy, it's name of vulners sub-project https://github.com/vulnersCom/vulners-proxy
so server_url can be the url of vulners-proxy

But proxy in ```vulners.VulnersApi(
            api_key=self.api_key,
            proxies=self.proxies,``` is a normal http-proxy